app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    @reviews = @video.reviews.all
```

Comment:
I see now, instead of `.all` I would prefer just `.reload`, but you should move it to `if` / `else` - to `else` because this way you have additional database query which is not necessary for happy path of `if @review.save`.